### ISSUE TITLE
Metrics "Short-circuit Read" and "Cache Hit Rate" for Alluxio Local does not update.

### ISSUE THREAD

**Issue by abhishek021** [Alluxio/alluxio#10110]
**Alluxio Version:**
Alluxio-2.0

**Describe the bug**
Metrics like short circuit read and Cache Hit Rate for Alluxio Local does not update
or more precisely it always gives zero on running `./alluxio fsadmin report metrics`.

So even if it's caching or not I don't know because it's updating other metrics like 
`Under Filesystem Read` and `Cache Hit Rate for Alluxio Remote`. 
But my Spark-UI shows PROCESS_LEVEL and NODE_LEVEL of data locality.

Note :- flag `alluxio.user.metrics.collection.enabled=true `is already there in alluxio-site.conf

**To Reproduce**
```
val df = spark.read.parquet("alluxio:///path/to/parquet/file")
df.agg(sum("XYZ")).show
```

**Expected behavior**
If my spark UI shows the data locality level at process-local and node-local then Alluxio
metric must show some short circuit read overall.

**Additional context**
To make sure of data-locality I'm running my setup with a single worker node and spark 
and alluxio-worker both running on that same node.

I also needed one clarification -

`spark.read.parquet("alluxio:///path/to/parquet/file")`
I think by the time spark does this transformation, it gathers node information where
it's going to deploy executors according to data locality. Is it so?

If not then when and how spark get to know the partition location on worker nodes?

**Comment by bf8086** [comment]
Hi @abhishek021. Thanks for report the issue.
Have you restarted Alluxio cluster after adding `alluxio.user.metrics.collection.enabled=true` to `alluxio-site.properties`?

**Comment by abhishek021** [comment]
Hi, @bf8086 Thanks for your response.
Yes, I restarted the cluster after adding the property.

**Comment by bf8086** [comment]
It's possible the property is not picked up by Alluxio client in Spark executors. Can you try adding `-Dalluxio.user.metrics.collection.enabled=true` to `spark.driver.extraJavaOptions` and `spark.executor.extraJavaOptions` in `spark-defaults.conf`?

**Comment by abhishek021** [comment]
Hi, @bf8086 I tried that too but no luck. Other metrics like Alluxio Remote Host and UFS read is being updated but not the Short-Circuit Read and Alluxio Local cache Hit.

**Comment by yuzhu** [comment]
@bf8086 any updates on this?

**Comment by bf8086** [comment]
It's likely a known issue previously reported as https://github.com/Alluxio/alluxio/issues/9273.

**Comment by jinhang** [comment]
I also encountered this problem, did you solve it?
Total IO:
    Short-circuit Read                                         0B
    Short-circuit Read (Domain Socket)                         0B
    From Remote Instances                                787.84GB
    Under Filesystem Read                                617.89GB
    Alluxio Write                                         32.60GB
    Alluxio Write (Domain Socket)                              0B
    Under Filesystem Write                                     0B
Total IO Throughput (Last Minute):
    Short-circuit Read                                         0B
    Short-circuit Read (Domain Socket)                         0B
    From Remote Instances                                  7.37KB
    Under Filesystem Read                                      0B
    Alluxio Write                                           2623B
    Alluxio Write (Domain Socket)                              0B
    Under Filesystem Write                                     0B
Cache Hit Rate (Percentage):
    Alluxio Local                                            0.00
    Alluxio Remote                                           100
    Miss                                                     0.00

**Comment by yuzhu** [comment]
fixed in 2.7.0
